src/course-outline/status-bar/StatusBar.jsx

```diff
@@ -3,15 +3,35 @@ import moment from 'moment/moment';
 import PropTypes from 'prop-types';
 import { FormattedDate, useIntl } from '@edx/frontend-platform/i18n';
 import {
-  Button, Hyperlink, Form, Stack,
+  Button, Hyperlink, Form, Sheet, Stack, useToggle,
 } from '@openedx/paragon';
 import { AppContext } from '@edx/frontend-platform/react';
 
+import { ContentTagsDrawer } from '../../content-tags-drawer';
+import TagCount from '../../generic/tag-count';
 import { useHelpUrls } from '../../help-urls/hooks';
 import { VIDEO_SHARING_OPTIONS } from '../constants';
 import messages from './messages';
 import { getVideoSharingOptionText } from '../utils';
 
+const StatusBarItem = ({ title, children }) => (
+  <div className="d-flex flex-column justify-content-between">
+    <h5>{title}</h5>
+    <div className="d-flex align-items-center">
+      {children}
+    </div>
+  </div>
+);
+
+StatusBarItem.propTypes = {
+  title: PropTypes.string.isRequired,
+  children: PropTypes.node,
+};
+
+StatusBarItem.defaultProps = {
+  children: null,
+};
+
```

Comment:
I created this component to avoid the repetition of the code below:

```javascript
<div className="d-flex flex-column justify-content-between">
        <h5>{title}</h5>
....
```